actors/miner/tests/change_beneficiary_test.rs

```diff
@@ -0,0 +1,411 @@
+use fil_actor_miner::BeneficiaryTerm;
+use fil_actors_runtime::test_utils::{expect_abort, MockRuntime};
+use fvm_shared::clock::ChainEpoch;
+use fvm_shared::{address::Address, econ::TokenAmount, error::ExitCode};
+use num_traits::Zero;
+
+mod util;
+use util::*;
+
+fn setup() -> (ActorHarness, MockRuntime) {
+    let big_balance = 20u128.pow(23);
+    let period_offset = 100;
+
+    let h = ActorHarness::new(period_offset);
+    let mut rt = h.new_runtime();
+    h.construct_and_verify(&mut rt);
+    rt.balance.replace(TokenAmount::from(big_balance));
+
+    (h, rt)
+}
+
+#[test]
+fn successfully_change_owner_to_another_address_two_message() {
```

Comment:
```suggestion
fn successfully_change_beneficiary_to_another_address_two_message() {
```

Reply:
for this test, beneficiary have two case
1. from owner address                                         need 2 message
2. from non owner address(also efficient)        need 3 message

Reply:
Oh, I see

`successfully_change_from_owner_...`